addons/website/static/tests/tours/link_tools.js

```diff
@@ -36,21 +36,21 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     // Remove the link.
     {
         content: "Click on the newly created link",
-        trigger: '.s_text_image a[href="http://odoo.com"]:contains("odoo.com")',
+        trigger: 'iframe #wrap .s_text_image a[href="http://odoo.com"]:contains("odoo.com")',
```

Comment:
Why is this change needed now while it wasn't before this PR?

Reply:
These are new steps, they were added in one of the forward-ported commits.